interviewing/readme.md

```diff
@@ -0,0 +1,66 @@
+# Interviewing 🧑‍💻
+
+Acing job interviews is a skill that can be learned like any other. Today we will pair up to practice interviewing.
```

Comment:
How long is this workshop? 60 minutes? 15 minutes per activity plus 5 minutes transition times?

Reply:
Yes, 60 minutes is what I was thinking:
- 15 minutes Stage 1: introduce yourself -  5 minutes craft intro, 2x2 minutes sharing intros, 5 minutes feedback
- 15 minutes Stage 2: soft skills questions - not expecting them to get through all the questions
- 15 minutes Stage 3: technical questions - goal here is to get them to start looking at technical questions, test their knowledge, and practice what they will say when they do not know an answer
- 15 minutes introducing the workshop, transition times and answering reflection questions at the end